packages/odyssey-react/src/components/CircularLoadIndicator/CircularLoadIndicator.theme.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  Size: theme.SpaceEmM,
+  TextColor: theme.ColorPrimaryBase,
+  CircleStrokeWidth: "17%",
+  CircleColor: theme.ColorPaletteNeutral100,
```

Comment:
This color isn't in use elsewhere yet. I believe it was a visual design choice, so I've left it in the zone of palette for now.